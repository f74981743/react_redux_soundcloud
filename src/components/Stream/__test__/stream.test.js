import React from 'react';
import ReactDOM from 'react-dom';
import { Stream } from '../index';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Component: Stream', () => {
    it('should render tracks', () => {
        const tree = toJson(shallow(<Stream tracks={[]} />));
        expect(tree).toMatchSnapshot();
    });
});