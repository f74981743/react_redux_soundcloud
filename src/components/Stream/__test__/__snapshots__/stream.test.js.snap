// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Stream should render tracks 1`] = `
<div
  className="tracks-board"
>
  <div
    className="container"
  >
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="fake-div"
    />
    <div
      className="loading"
    />
  </div>
</div>
`;
